.github/workflows/update-s3-and-cloudfront.yaml

```
@@ -0,0 +1,36 @@
+name: Update S3 and Cloudfront
+on:
+  push:
+    branches:
+      - main
```

```suggestion
on:
  push:
    branches:
      - main
    paths:
      - .github/workflows/update-s3-and-cloudfront.yaml
      - s3/install
```

Maybe we could trigger the upload when only two specific files in a push are modified?

---

Sure. This was similar to how we run this on the docs repo.